frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelWelcome.tsx

```diff
@@ -0,0 +1,250 @@
+                            />
+                        </div>
+                    </Card>
+                    <Card>
+                        <Title>Notebooks</Title>
+                        <Description>
+                            Analyze data from different angles and share results with your team - all in
+                            a&nbsp;single&nbsp;document.
+                        </Description>
+                        <div className="-mr-4 -mb-3">
+                            <Image
+                                src={isDarkModeOn ? featureNotebooksDark : featureNotebooks}
+                                alt="Notebooks"
+                                width="100%"
+                                style={{ borderTopLeftRadius: '0.25rem' }}
+                            />
+                        </div>
+                    </Card>
+                </Row>
+
+                <Row>
+                    <Card>
+                        <div className="grid grid-cols-2 gap-5 items-center">
+                            <div>
+                                <Title>Side panel</Title>
+                                <Description>
+                                    It's this multipurpose thing you're looking at right&nbsp;now!
+                                </Description>
+                                <Description>Create notebooks, read docs, contact support, and&nbsp;more.</Description>
+                            </div>
+                            <div className="-mr-4 -my-3">
+                                <Image
+                                    src={isDarkModeOn ? featureSidePanelDark : featureSidePanel}
+                                    alt="Side panel"
+                                    height="100%"
+                                    style={{ maxHeight: 205 }}
+                                />
+                            </div>
+                        </div>
+                    </Card>
+                </Row>
+
+                <Row className="grid grid-cols-2" columns="calc(60% - 1.25rem) 40%">
+                    <Card>
+                        <Title>Improved search</Title>
+                        <Description>
+                            Search for anything with <KeyboardShortcut command k />
+                        </Description>
+                        <div className="-mr-4 -mb-3">
+                            <Image
+                                src={isDarkModeOn ? featureSearchDark : featureSearch}
+                                alt="Improved search"
+                                width="100%"
+                                style={{ borderTopLeftRadius: '0.25rem' }}
+                            />
+                        </div>
+                    </Card>
+                    <Card className="flex flex-col">
+                        <div className="flex-1">
+                            <Title>Command bar</Title>
```

Comment:
The "command palette" is the old name… the new iteration is called the "command bar"